<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramesView</class>
 <widget class="QWidget" name="FramesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="animationPreview">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </widget>
  <widget class="QSlider" name="fPSSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="animationCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Animate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFrameButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFrameButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyFrameButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="framesScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>349</width>
      <height>139</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="framesDisplayArea"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
